### PR TITLE
DOCS-15299 fips mode legacy shell install

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
https://jira.mongodb.org/browse/DOCS-15299

### Stage

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCS-15299-fips-mode-legacy-shell-install/install/#install-mongosh-1

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
This one LGTM, @jason-price-mongodb!

Thanks,
Joe

**Comment by jason-price-mongodb** [comment]
@mmarcon please review. Thanks.

File: source/includes/steps-install-shell-linux-rpm.yaml [hunk 1]
```diff
@@ -69,4 +69,11 @@ content: |
   .. code-block:: sh
 
     sudo yum install -y mongodb-mongosh
+  
+  To install ``mongosh`` that supports OpenSSL, issue the following
```
[the remark this answers is not in the document]
Comment (mmarcon) [reply source/includes/steps-install-shell-linux-rpm.yaml:73]: It's not that there is a version of `mongosh` that supports OpenSSL and one that does not support it. The difference is more subtle: the regular `mongosh` binary includes a vendored copy of OpenSSL, while the binaries installed by the`mongodb-mongosh-shared-openssl11` and `mongodb-mongosh-shared-openssl3` packages use the OpenSSL library distributed with the operating system (versions 1.1 and 3 respectively).

[the remark this answers is not in the document]
Comment (jason-price-mongodb) [reply source/includes/steps-install-shell-linux-rpm.yaml:73]: Done.

**Comment by jason-price-mongodb** [comment]
@mmarcon please re-view. Thanks.

**Review by mmarcon — CHANGES_REQUESTED** [review]

File: source/includes/steps-install-shell-linux-rpm.yaml [hunk 2]
```diff
@@ -63,10 +63,41 @@ level: 4
...
+  .. code-block:: sh 
+
+     sudo yum install -y mongodb-mongosh-shared-openssl3
+
+  To install ``mongosh`` that uses OpenSSL 3.0 on Ubuntu, issue
+  the following command:
+
+  .. code-block:: sh 
+
+     sudo apt-get install -y mongodb-mongosh-shared-openssl3
```
Comment (mmarcon) [inline source/includes/steps-install-shell-linux-rpm.yaml:102]: I think there is some confusion here.

```
sudo apt-get install -y mongodb-mongosh-shared-openssl11
sudo apt-get install -y mongodb-mongosh-shared-openssl3
```
is what you do on distributions that use `deb` packages (Ubuntu and Debian) and therefore those instructions should be in the `deb` tab in the page.

```
sudo yum install -y mongodb-mongosh-shared-openssl11
sudo yum install -y mongodb-mongosh-shared-openssl3
```
is what you do on distributions that use `deb` packages (Ubuntu and Debian) and therefore those instructions should be in the `rpm` tab in the page.

For `tar.gz` we need to point people to downloading the `tar.gz` with shared openssl library from the download center and installing that as an alternative to e.g. `mongosh-1.5.0-linux-x64.tgz`.

**Review by mmarcon — CHANGES_REQUESTED** [review]

File: source/includes/steps-install-shell-linux-rpm.yaml (hunk 2 above)
Comment (jason-price-mongodb) [reply source/includes/steps-install-shell-linux-rpm.yaml]: Done.

**Comment by jason-price-mongodb** [comment]
@mmarcon please re-review. Thanks.

File: source/includes/steps-install-shell-linux-generic.yaml [hunk 3]
```diff
@@ -7,6 +7,12 @@ ref: navigate-dlc-generic-linux
 title: "Download the Linux 64-bit ``.tgz`` package."
 ref: download-archive-rhel
 level: 4 
+content: |
+
+   Download the appropriate version of ``mongosh`` for your operating
+   system. MongoDB also provides versions of ``mongosh`` that support
```
Comment (mmarcon) [inline source/includes/steps-install-shell-linux-generic.yaml:13]:
```suggestion
   system. MongoDB also provides versions of ``mongosh`` that uses the system's
```

**Review by mmarcon — APPROVED** [review]
I suggested a minor change. Other than that, it's good to go.

File: source/includes/steps-install-shell-linux-generic.yaml (hunk 3 above)
Comment (jason-price-mongodb) [reply source/includes/steps-install-shell-linux-generic.yaml]: Done.

**Pull request merged**